tests/test.py

```diff
@@ -138,12 +138,12 @@ def test_predict_response():
     response = r.json()
 
     assert response['status'] == 'ok'
-    assert round(float(response['predictions'][0][0]['toxic'])) == 1
-    assert round(float(response['predictions'][0][0]['severe_toxic'])) == 1
-    assert round(float(response['predictions'][0][0]['obscene'])) == 1
-    assert round(float(response['predictions'][0][0]['insult'])) == 1
-    assert round(float(response['predictions'][0][0]['threat'])) == 1
-    assert round(float(response['predictions'][0][0]['identity_hate'])) == 1
+    assert round(float(response['predictions'][0]['toxic'])) == 1
+    assert round(float(response['predictions'][0]['severe_toxic'])) == 1
+    assert round(float(response['predictions'][0]['obscene'])) == 1
+    assert round(float(response['predictions'][0]['insult'])) == 1
+    assert round(float(response['predictions'][0]['threat'])) == 1
+    assert round(float(response['predictions'][0]['identity_hate'])) == 1
```

Comment:
I'm not sure how much this matters, but I would suggest using adjective for the last three items as well (insulting, threatening, identity_hating)

Reply:
Is there a "standard" or commonly used definition for the different categories?

Reply:
Not a "standard" but the labels match the dataset: https://www.kaggle.com/c/jigsaw-toxic-comment-classification-challenge/data

Reply:
I agree we should keep this if the underlying dataset does so.